backend/src/main/java/heartbeat/controller/report/dto/response/ReportResponse.java

```diff
@@ -33,11 +33,13 @@ public class ReportResponse {
 
 	private Long exportValidityTime;
 
-	private boolean boardMetricsCompleted;
+	private Boolean boardMetricsCompleted;
 
-	private boolean doraMetricsCompleted;
+	private Boolean doraMetricsCompleted;
 
-	private boolean allMetricsCompleted;
+	private Boolean overallMetricsCompleted;
+
+	private Boolean allMetricsCompleted;
```

Comment:
remove `allMetricsCompleted`,  and check in the frontend side

Reply:
we discussion, still plan to put the logic on the backend, and won't change it for now，do you think？